<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWind</class>
 <widget class="QMainWindow" name="QWMainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMainWind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="txtEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actLose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actCopy"/>
    <addaction name="actCut"/>
    <addaction name="actPaste"/>
    <addaction name="separator"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actBold"/>
    <addaction name="actFontItalic"/>
    <addaction name="actFontUnderLine"/>
    <addaction name="separator"/>
    <addaction name="actToolbarLab"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="actOpen"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actBold"/>
   <addaction name="actFontItalic"/>
   <addaction name="actFontUnderLine"/>
   <addaction name="separator"/>
   <addaction name="actFont"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actCut">
   <property name="icon">
    <iconset theme="w" resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/cut.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/20211127084526.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/20211127084526.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/paste.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="粗体" resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/BLD.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/BLD.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="actFontItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/2021104113643.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/2021104113643.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actFontUnderLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/WORDUNDR.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/WORDUNDR.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actLose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/103.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/103.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/120.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/120.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/20.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/20.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset>
     <activeon>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/2021104113447.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/new.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="actToolbarLab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/WORDUNDR.png</normaloff>:/image/D:/GAME_yuanshen/Genshin Impact Game/ScreenShot/WORDUNDR.png</iconset>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actLose</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
